src/plotting/plotSettings/yAxisTab.component.tsx

```diff
@@ -231,6 +238,23 @@ const YAxisTab = (props: YAxisTabProps) => {
     ]
   );
 
+  const changeAxisLabel = React.useCallback(
```

Comment:
There's no unit test covering this new function - hence why the patch coverage is failing